packages/app/features/account/settings/backup/index.tsx

```diff
@@ -157,7 +157,7 @@ const SendAccountCredentials = ({
       w={'100%'}
       gap={'$size.1.5'}
       p={'$size.1.5'}
-      $theme-dark={{ backgroundColor: '$darkest' }}
+      $theme-dark={{ backgroundColor: '$black' }}
       $theme-light={{ backgroundColor: '$gray2Light' }}
```

Comment:
```
$theme-dark={{ backgroundColor: '$darkest' }}
$theme-light={{ backgroundColor: '$gray2Light' }}
```
There should be a radix color for this now. I think its this
```suggestion
      bc={"$color1"}
```

Reply:
Nice yeah that worked, that's super handy to know.

Reply:
Yeah major QOL upgrade